packages/babel-parser/typings/babel-parser.d.ts

```diff
@@ -94,174 +95,136 @@ interface TypeScriptPluginOptions {
-   */
-  sourceType?: "script" | "module" | "unambiguous";
-
-  /**
-   * Correlate output AST nodes with their source filename.
-   * Useful when generating code and source maps from the ASTs of multiple input files.
-   */
-  sourceFilename?: string;
-
-  /**
-   * By default, all source indexes start from 0.
-   * You can provide a start index to alternatively start with.
-   * Useful for integration with other source tools.
-   */
-  startIndex?: number;
-
-  /**
-   * By default, the first line of code parsed is treated as line 1.
-   * You can provide a line number to alternatively start with.
-   * Useful for integration with other source tools.
-   */
-  startLine?: number;
-
-  /**
-   * By default, the parsed code is treated as if it starts from line 1, column 0.
-   * You can provide a column number to alternatively start with.
-   * Useful for integration with other source tools.
-   */
-  startColumn?: number;
-
-  /**
-   * Array containing the plugins that you want to enable.
-   */
-  plugins?: ParserPlugin[];
-
-  /**
-   * Should the parser work in strict mode.
-   * Defaults to true if sourceType === 'module'. Otherwise, false.
-   */
-  strictMode?: boolean;
-
-  /**
-   * Adds a ranges property to each node: [node.start, node.end]
-   */
-  ranges?: boolean;
-
-  /**
-   * Adds all parsed tokens to a tokens property on the File node.
-   */
-  tokens?: boolean;
-
-  /**
-   * By default, the parser adds information about parentheses by setting
-   * `extra.parenthesized` to `true` as needed.
-   * When this option is `true` the parser creates `ParenthesizedExpression`
-   * AST nodes instead of using the `extra` property.
-   */
-  createParenthesizedExpressions?: boolean;
-
-  /**
-   * The default is false in Babel 7 and true in Babel 8
-   * Set this to true to parse it as an `ImportExpression` node.
-   * Otherwise `import(foo)` is parsed as `CallExpression(Import, [Identifier(foo)])`.
-   */
-  createImportExpressions?: boolean;
-}
-
-type ParserPlugin = PluginConfig;
-
+declare function parse(input: string, options?: ParserOptions): ParseResult<File>;
+declare function parseExpression(input: string, options?: ParserOptions): ParseResult<Expression>;
 
 declare const tokTypes: {
   // todo(flow->ts) real token type
   [name: string]: any;
 };
 
-interface ParseError {
-  code: string;
-  reasonCode: string;
-}
-
-type ParseResult<Result> = Result & {
-  errors: ParseError[];
-};
-
-export { DecoratorsPluginOptions, FlowPluginOptions, ParseError, ParseResult, ParserOptions, ParserPlugin, ParserPluginWithOptions, PipelineOperatorPluginOptions, RecordAndTuplePluginOptions, TypeScriptPluginOptions, parse, parseExpression, tokTypes };
+export { DecoratorsPluginOptions, FlowPluginOptions, ParseError, ParseResult, ParserOptions, PluginConfig as ParserPlugin, ParserPluginWithOptions, PipelineOperatorPluginOptions, RecordAndTuplePluginOptions, TypeScriptPluginOptions, parse, parseExpression, tokTypes };
```

Comment:
Note: the difference in these two lines is that before it contained `, ParserPlugin,`, while after it contains `, PluginConfig as ParserPlugin,`.